src/phases/form-change-phase.ts

```diff
@@ -197,6 +199,17 @@ export class FormChangePhase extends FormChangeBasePhase {
   public override end(): void {
     const { ui } = globalScene;
 
+    /**
+     * Edgecase
+     *
+     * If a fused Pokemon has either of its halves change form then it will attempt to learn the
+     * EVOLVE_MOVE of both halves of the fusion
+     */
+    const formChangeLearnMove = this.pokemon.getLevelMoves(EVOLVE_MOVE, true);
+    for (const lm of formChangeLearnMove) {
+      globalScene.unshiftPhase(new LearnMovePhase(globalScene.getPlayerParty().indexOf(this.pokemon), lm[1]));
+    }
```

Comment:
NIT: use array destructure
```suggestion
    for (const [ , secondMoveToLearn ] of formChangeLearnMove) {
      globalScene.unshiftPhase(new LearnMovePhase(globalScene.getPlayerParty().indexOf(this.pokemon), secondMoveToLearn]));
    }
```

Reply:
I can also make this change in the `level-up-phase` where I stole it from